assets/js/keyBindings.coffee

```diff
@@ -87,7 +87,8 @@ if module?
     ENTER_VISUAL      : ['v']
     ENTER_VISUAL_LINE : ['V']
 
-    EXPORT            : ['ctrl+s']
+    EXPORT_FILE       : ['ctrl+s'] # Really bad shortcuts, change
```

Comment:
Suggestions for better import/export shortcuts?

Also why do you want these included in insert mode?